composeApp/src/main/kotlin/io/github/couchtracker/ui/components/CrewCompactListItem.kt

```diff
@@ -106,3 +128,15 @@ suspend fun List<TmdbCrew>.toCrewCompactListItemModel(
         }
     }.awaitAll()
 }
+
+@JvmName("TmdbAggregateCrew_toCrewCompactListItemModel")
+suspend fun List<TmdbAggregateCrew>.toCrewCompactListItemModel(
```

Comment:
Question: are we sorting the crew in any way? I don't see it, but it looks like the order is pretty arbitrary, and it looks like "important" people are not at the start.

Reply:
Tmdb itself doesn't seem to order them either, so I think we can leave as-is